### ISSUE TITLE
Compose different frontend views into one

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#120]
## User story
1. As a user
2. I want a composed view of different FE components
3. So that the FE looks good and is intuitive

## Acceptance criteria
* Integrate different views into one panel
* UI elements are completed (e.g.: all tabs have names not numbers in brakets, etc.)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
